nengo/__init__.py

```diff
@@ -26,11 +26,13 @@
 from .learning_rules import PES, BCM, Oja, Voja
 from .params import Default
 from .probe import Probe
+from .processes import (BrownNoise, FilteredNoise, PresentInput, Process,
+                        WhiteNoise, WhiteSignal)
```

Comment:
Not sure how much sense it makes to have only parts of a module as core part. If I have to think about whether I can use a specific class with `nengo.Foo` or it has to be `nengo.dists.Foo`/`nengo.processes.Foo`, I'd rather always type `nengo.dists.Foo`/`nengo.processes.Foo`.